pkg/controller/pod_driver.go

```diff
@@ -699,29 +699,26 @@ func (p *PodDriver) CleanUpCache(ctx context.Context, pod *corev1.Pod) {
 	}
 }
 
-func (p *PodDriver) OverwirteMountPodResourcesWithPVC(ctx context.Context, pod *corev1.Pod) error {
-	pvName := pod.Annotations[config.UniqueId]
-	pv, err := p.Client.GetPersistentVolume(ctx, pvName)
+func (p *PodDriver) applyConfigPatch(ctx context.Context, pod *corev1.Pod) error {
+	attr, err := config.GenPodAttrWithMountPod(ctx, p.Client, pod)
 	if err != nil {
-		klog.Errorf("Get pv %s error: %v", pvName, err)
 		return err
 	}
-	pvc, err := p.Client.GetPersistentVolumeClaim(ctx, pv.Spec.ClaimRef.Name, pv.Spec.ClaimRef.Namespace)
-	if err != nil {
-		klog.Errorf("Get pvc %s/%s error: %v", pv.Spec.ClaimRef.Namespace, pv.Spec.ClaimRef.Name, err)
-		return err
-	}
-
-	cpuLimit := pvc.Annotations[config.MountPodCpuLimitKey]
-	memoryLimit := pvc.Annotations[config.MountPodMemLimitKey]
-	cpuRequest := pvc.Annotations[config.MountPodCpuRequestKey]
-	memoryRequest := pvc.Annotations[config.MountPodMemRequestKey]
-
-	resources, err := config.ParsePodResources(cpuLimit, memoryLimit, cpuRequest, memoryRequest)
-	if err != nil {
-		return fmt.Errorf("parse pvc resources error: %v", err)
-	}
-	pod.Spec.Containers[0].Resources = resources
+	// update pod spec
+	pod.Labels = attr.Labels
```

Comment:
does this mean, all below fields can be customized in configmap, and updated dynamically? (only need to re-create mount pod)

Reply:
yep